Branch-TestBed/Branch-TestBed/AppDelegate.m

```diff
@@ -35,6 +35,31 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     [branch setDeepLinkDebugMode:@{@"example_debug_param" : @"foo"}];
     
+    NSString *webViewHtml = @"<!DOCTYPE html><html><body><h1>Branch View Test</h1><p>Branch View Test.</p>\n\n\n <a class=\"accept_btn\" href=\"branch-cta://accept\">Accept</a>\n\n<a class=\"cancel_btn\" href=\"branch-cta://cancel\">Cancel</a></body></html>";
+    
+    NSMutableArray * branchViewArray = [[NSMutableArray alloc] init];
+    NSDictionary * branchViewItem1 = [NSDictionary dictionaryWithObjectsAndKeys:
+                                     @"branch_view_id_01", @"branch_view_id",
+                                     @"open", @"branch_view_action",
+                                     @"1", @"num_of_use",
+                                     webViewHtml, @"branch_view_html",
+                                     @"1489176401000", @"expiry",
+                                     @"true", @"debug",
+                                     nil];
+    NSDictionary * branchViewItem2 = [NSDictionary dictionaryWithObjectsAndKeys:
+                                     @"branch_view_id_02", @"branch_view_id",
+                                     @"buy", @"branch_view_action",
+                                     @"3", @"num_of_use",
+                                     webViewHtml, @"branch_view_html",
+                                     @"1489176401000", @"expiry",
+                                     @"true", @"debug",
+                                     nil];
+  
+    [branchViewArray addObject:branchViewItem1];
+    [branchViewArray addObject:branchViewItem2];
+    [branch setDeepLinkDebugMode:@{@"branch_view_data" : branchViewArray }];
```

Comment:
if it's not too much work, could you make a separate method that returns branchViewArray? That will keep the app delegate much cleaner.

Also, I'm not sure we want this presented in the testbed every time, so you could have this one line, `[branch setDeepLinkDebugMode:@{@"branch_view_data" : branchViewArray }];` be commented out with a note that users should uncomment it out to test it. Or perhaps @aaustin would prefer to have this enabled by default.